### DISCUSSION TITLE
Make link types orderable

### DISCUSSION THREAD

**Discussion by ryanleichty** [craftcms/cms#15659] [Ideas] [answered]
It would be great if you could reorder the link types available in link fields, similar to how you can with the Hyper plugin by Verbb.

We typically reorder the link types based on the assumed usage of the field. That way you have less clicking if 95% of the time you want to select an entry.

![CleanShot 2024-09-04 at 09 03 14@2x](https://github.com/user-attachments/assets/bc0feeac-a07a-40fe-99e0-e36e295af586)

**Comment by mmikkel** [comment]
I recently posted a FR for the ability to set a _default link type_ per field (https://github.com/craftcms/cms/discussions/15516), which I think would also solve the described use case, for the most part.

Orderable link types would definitely be a good solve as well, though – I'd be super happy to see either solution implemented 🙂

**Reply by ryanleichty** [reply]
Nice, that would solve it as well!

**Reply by mmikkel** [reply]
Yeah, the primary thing for me is being able to put "Entry" at the top of the dropdown, because in the vast majority of cases that's the most commonly used option (or, the option that we as developers want to push the author towards), in fields that support both entries and custom/external URLs. Having the types orderable, or being able to set a default, would both solve that nicely.

**Reply by brandonkelly** [reply]
@mmikkel Internally I put “Sortable link types” as the planned solution for #15516 :)

**Comment by brandonkelly** [comment, marked as answer]
We’ve made the Allowed Link Types setting sortable for Craft 5.5 :tada: (#15963)

**Reply by brandonkelly** [reply]
Craft 5.5.0 is out with that change 🎉
